templates/fine-tune-llm_v2/cookbooks/optimize_cost/README.ipynb

```diff
@@ -70,7 +70,7 @@
     "Then you want to run the template entry-point and run the job for a few iterations to note down the time it takes for different steps. In the logs, we print a table with what you need, but you also can go to WANDB and obtain the metrics from there. \n",
     "\n",
     "```\n",
-    "python main.py <path_to_config.yaml>\n",
+    "llmforge anyscale finetune <path_to_config.yaml>\n",
```

Comment:
These same changes should also be done for the cookbooks: 
1. https://github.com/anyscale/templates/tree/main/templates/fine-tune-llm_v2/cookbooks/lora_vs_full_parameter 
2. https://github.com/anyscale/templates/tree/main/templates/fine-tune-llm_v2/cookbooks/continue_from_checkpoint